test/test_game.py

```diff
@@ -0,0 +1,221 @@
+import unittest
+
+from rock_paper_scissors.core.exc import InvalidRoundsError, InvalidChoiceError
+from rock_paper_scissors.core.game import Game
+from rock_paper_scissors.core.moves import Paper, Rock
+from rock_paper_scissors.core.user import User
+from rock_paper_scissors.core.valid_choices import VALID_CHOICES
+
+
+class GameInitTestCase(unittest.TestCase):
+    def test_player1_and_player2_and_starting_rounds_types(self):
+        player1 = 'Joe'
+        player2 = 'Jane'
+        game = Game(player1, player2)
+
+        self.assertIsInstance(game.player1, str)
+        self.assertIsInstance(game.player2, str)
+        self.assertIsInstance(game._rounds, int)
+        self.assertEqual(game._rounds, 0)
+
+
+class GameSetRoundsTestCase(unittest.TestCase):
+    def setUp(self):
+        self.game = Game('player1', 'player2')
+
+    def test_valid_rounds_input(self):
+        value = self.game.rounds = 11
+
+        self.assertIsInstance(value, int)
```

Comment:
Instead of testing type directly, as well as testing that the value is greater than 0, I would simply do this:
```python
self.assertEqual(value, 11)
```

This will test both criteria implicitly, but will also explicitly test that `self.game.rounds` is set properly when given a valid integer value.

Reply:
I hadn't thought about testing if a value was properly set.